static/src/javascripts/projects/common/modules/commercial/build-page-targeting.spec.js

```diff
@@ -75,6 +84,15 @@ const tcfMixedConsentMock = (callback): void =>
 const ccpaWithConsentMock = (callback): void => callback(false);
 const ccpaWithoutConsentMock = (callback): void => callback(true);
 
+const tcfv2WithConsentMock = (callback): void =>
+    callback({ tcfv2 : { tcfData: {'1': true, '2': true, '3': true, '4': true, '5': true }}});
+const tcfv2WithoutConsentMock = (callback): void =>
+    callback({ tcfv2 : { tcfData: { '1': false, '2': false, '3': false, '4': false, '5': false }}});
```

Comment:
In this case, I think the real data would be the empty object `{}`

Reply:
We have decided that our wrapper should always return an object with all purposes set to a boolean, see https://github.com/guardian/consent-management-platform/pull/190